packages/react-native/ReactCommon/react/renderer/components/view/conversions.h

```diff
@@ -556,6 +556,36 @@ inline void fromRawValue(
   result = transformMatrix;
 }
 
+ inline void fromRawValue(
+     const PropsParserContext &context,
+     const RawValue &value,
+     TransformOrigin &result) {
+   react_native_expect(value.hasType<std::vector<RawValue>>());
+   auto origins = (std::vector<RawValue>)value;
+   
+   TransformOrigin transformOrigin;
+
+   size_t i = 0;
+   for (const auto& origin : origins) {
+     if (origin.hasType<Float>()) {
```

Comment:
Is this just a stripped back version of https://github.com/facebook/react-native/blob/35badafd5f6ca1bc25ca0a91101e46aa828671f2/packages/react-native/ReactCommon/react/renderer/components/view/conversions.h#L406 ? It might be ok to just call that function and just be careful in `getTranslateForTransformOrigin` to ignore anything with `YGValueUndefined` (which covers `auto`)

Reply:
> Is this just a stripped back version of

yeah kind of. We set TransformOrigin in the result and we have a vector in the raw value. also now it adds some check like origins should be of size 3

Reply:
>  It might be ok to just call that function and just be careful in getTranslateForTransformOrigin to ignore anything with the value YGUndefined (which covers auto)

yeah, right now it'll only handle point and percent. https://github.com/facebook/react-native/pull/38559/commits/302a9aaf283a2d05de01ea314e996fb848c89bcb